ci-operator/step-registry/hypershift/mce/agent/metal3/conformance/hypershift-mce-agent-metal3-conformance-workflow.yaml

```diff
@@ -0,0 +1,34 @@
+workflow:
+  as: hypershift-mce-agent-metal3-conformance
+  documentation: |-
+    The HyperShift mce agent conformance (connected) workflow executes tests against a new ephemeral
+    The installation method for nodepool is metal3.
+    The current workflow supports both IPv4 and dual-stack
+  steps:
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    post:
+    - chain: hypershift-mce-agent-destroy
+    - chain: baremetalds-ofcir-post
+    test:
+    - chain: hypershift-conformance
+    pre:
+    - chain: baremetalds-ofcir-pre
```

Comment:
This is the typo I mentioned in the overall review comment

Reply:
baremetalds is as expected, where 'ds' stands for https://github.com/openshift-metal3/dev-scripts

Reply:
ohh, gotcha, It's fine then